### PR TITLE
[Added]: Test cases for table graph

### PR DESCRIPTION
**Pull request by nxanil**
Added test cases for `Table graph`

**Results:** 
 PASS  src/lib/vis-graphs/Graphs/Table/table.spec.js
  Table Graph
    Table
      Hide SearchBar
        ✓ Total Column (4ms)
        ✓ Column Names (2ms)
        ✓ First Row Data (2ms)
        ✓ Second Row Data (2ms)
        ✓ Total Rows (1ms)
        ✓ SearchBar Text
        ✓ SearchBar (1ms)
      selectAll
        ✓ Total Column (1ms)
        ✓ Column Names (1ms)
        ✓ First Row Data (2ms)
        ✓ Second Row Data (2ms)
        ✓ Total Rows (1ms)
      highlightColumn
        ✓ Total Column
        ✓ Column Names (1ms)
        ✓ First Row Data (1ms)
        ✓ Second Row Data (2ms)
        ✓ Total Rows
      selectColumn
        ✓ Total Column (1ms)
        ✓ Column Names (1ms)
        ✓ First Row Data (1ms)
        ✓ Second Row Data (1ms)
        ✓ Total Rows (1ms)
        ✓ Select Column DropDown (1ms)
      toolTip
        ✓ Total Column (1ms)
        ✓ Column Names
        ✓ First Row Data (2ms)
        ✓ Second Row Data (1ms)
        ✓ Total Rows (1ms)
      Click Event
        ✓ Click On Table (3ms)

Test Suites: 1 passed, 1 total
Tests:       29 passed, 29 total

### CODE REVIEW

**Comment by nxanil** [comment]
@preshitg 

can you please review this PR.

File: Graphs/Table/index.js [hunk 1]
```diff
@@ -731,8 +731,8 @@ const TableGraph = (props) => {
                                                     setCurrentStartIndex(props.startIndex);
                                                     onRowsRendered(props);
                                                 }}
-                                                width={graphWidth}
-                                                height={height}
+                                                width={graphWidth || 500}
```
Comment (preshitg) [inline Graphs/Table/index.js:734]: Why do we need this change? This is going to affect all Tables.
We should not modify source code in test cases PR.

Comment (nxanil) [reply Graphs/Table/index.js]: Changes done.

**Comment by nxanil** [comment]
@preshitg

Feedback changes done. Can you please review this PR.

File: Graphs/testHelper.js [hunk 2]
```diff
@@ -61,24 +61,25 @@ export const checkSvg = (component) => {
...
     return noOfRows;
 }
 
 export const totalColumn = ($) => {
-    const noOfColumns = $('table').find('thead').find('tr').children().length;
+    const noOfColumns = $('.ReactVirtualized__Table').attr('aria-colcount');
     return noOfColumns;
 }
 
 export const checkRowData = ($, rowNo) => {
```
Comment (preshitg) [inline Graphs/testHelper.js:73]: Check my comment in the other PR: https://github.com/nuagenetworks/vis-graphs/pull/462#discussion_r627713223
We should not pass hard-coded indices, instead compute the object in the test and pass the object. That way we can reuse this function without having to add multiple if statements.

Comment (nxanil) [reply Graphs/testHelper.js]: Done.

**Comment by nxanil** [comment]
@preshitg

Feedback changes done. Can you please review this PR.

**Review by preshitg — APPROVED** [review]

**Pull request merged**